shashi/jan29.js

```diff
@@ -0,0 +1,29 @@
+//Create a function that moves all capital letters to the front of a word
+//Assumptions:
+//Input string only
+
+function capToFront(str) {
+  var caps = "";
+  var nonCaps = "";
+  for (let i = 0; i < str.length; i++) {
+    str[i] !== str[i].toLowerCase() ? caps += str[i] : nonCaps += str[i];
```

Comment:
this works, but you can do something much easier. you can just check for the ranges A to Z